radio/util/tts_de.py

```diff
@@ -8,11 +8,14 @@
 systemSounds = []
 sounds = []
 
-for i in range(101):
+for i in range(100):
     systemSounds.append((str(i), filename(PROMPT_SYSTEM_BASE + i)))
-systemSounds.append(("tausend", filename(PROMPT_SYSTEM_BASE + 101)))
-for i, s in enumerate(["comma", "und", "minus", "uhr", "minute", "minuten", "sekunde", "sekunden"]):
-    systemSounds.append((s, filename(PROMPT_SYSTEM_BASE + 102 + i)))
+systemSounds.append(("ein", filename(PROMPT_SYSTEM_BASE + 100)))
+systemSounds.append(("eine", filename(PROMPT_SYSTEM_BASE + 101)))
+systemSounds.append(("hundert", filename(PROMPT_SYSTEM_BASE + 102)))
+systemSounds.append(("tausend", filename(PROMPT_SYSTEM_BASE + 103)))
+for i, s in enumerate(["komma", "und", "minus", "uhr", "minute", "minuten", "sekunde", "sekunden", "stunde", "stunden"]):
```

Comment:
did you check that komma really sounds better than comma? The mispelling are sometimes to get better pronounciation.